### PR TITLE
Authentication Feature integrating NuxtAuth, Pinia and Vuelidate

### PR DESCRIPTION
**Pull request by KaelNierras**

### CODE REVIEW

File: composables/useValidation.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    email: { required, email },
+    password: { required, minLength: minLength(6) },
+    password_confirmation: { required, sameAsPassword: sameAs(() => formRegister.value.password) },
+  };
+  
+  const v$Login = useVuelidate(LoginRules, LoginForm);
+  const v$Register = useVuelidate(RegisterRules, RegisterForm);
+
+  return { LoginForm, v$Login, RegisterForm, v$Register };
+}
```
Comment (bry221996) [inline composables/useValidation.ts]: hi @KaelNierras 

lets not add this form values and form validations here.

you can pass this from the specific component that will use this .

File: pages/authentication/login.vue
Comment (bry221996) [inline pages/authentication/login.vue:1]: @KaelNierras 

can we make the routes


/login
/register 

?

File: pages/authentication/verification.vue
Comment (bry221996) [inline pages/authentication/verification.vue:1]: lets remove this file for now. since we dont have verification yet :)

File: pages/index.vue
Comment (bry221996) [inline pages/index.vue:1]: lets add some text here. just for viewing

**Comment by KaelNierras** [comment]
Revision Resolve: Please check new commits

**Pull request merged**